src/randoop/FailureSet.java

```diff
@@ -6,35 +6,56 @@
 import java.util.Set;
 
 import randoop.operation.MethodCall;
-import randoop.operation.Operation;
 import randoop.sequence.ExecutableSequence;
 import randoop.sequence.MutableSequence;
 import randoop.sequence.MutableVariable;
+import randoop.sequence.Statement;
 import randoop.sequence.Variable;
 
+/**
+ * FailureSet represents a set of failures that occurred in the execution of a particular
+ * {@link ExecutableSequence}. 
+ *
+ */
 public class FailureSet {
   
   private Set<Failure> failures = new LinkedHashSet<Failure>();
   
+  /**
+   * FailureSet.Failure represents an execution failure and holds the
+   * statement and class where the failure occurred.
+   *
+   */
   public static class Failure {
-    public final Operation st;
-    public final Class<?> viocls;
-    public Failure(Operation st, Class<?> viocls) {
-      this.st = st;
-      this.viocls = viocls;
+    
+    /**
```

Comment:
It's up to you, but for one-line Javadoc comments I put them all on one line:

    /** .... */

rather than

    /**
     * ...
     */